x-pack/plugins/ml/public/application/explorer/explorer.tsx

```diff
@@ -344,208 +449,258 @@ export const Explorer: FC<ExplorerUIProps> = ({
       queryString={queryString}
       updateLanguage={updateLanguage}
     >
-      <div className="results-container">
-        {noInfluencersConfigured && (
-          <div className="no-influencers-warning">
-            <EuiIconTip
-              content={i18n.translate('xpack.ml.explorer.noConfiguredInfluencersTooltip', {
-                defaultMessage:
-                  'The Top Influencers list is hidden because no influencers have been configured for the selected jobs.',
-              })}
-              position="right"
-              type="iInCircle"
-            />
-          </div>
-        )}
-
-        {noInfluencersConfigured === false && (
-          <div className="column col-xs-2" data-test-subj="mlAnomalyExplorerInfluencerList">
-            <EuiSpacer size={'s'} />
-            <EuiTitle className="panel-title">
-              <h2>
-                <FormattedMessage
-                  id="xpack.ml.explorer.topInfuencersTitle"
-                  defaultMessage="Top influencers"
-                />
-                <EuiIconTip
-                  content={
-                    <FormattedMessage
-                      id="xpack.ml.explorer.topInfluencersTooltip"
-                      defaultMessage="View the relative impact of the top influencers in the selected time period and add them as filters on the results. Each influencer has a maximum anomaly score between 0-100 and a total anomaly score for that period."
+      <EuiResizableContainer
+        direction={isMobile ? 'vertical' : 'horizontal'}
+        onPanelWidthChange={onPanelWidthChange}
+      >
+        {(EuiResizablePanel, EuiResizableButton, actions) => {
+          collapseFn.current = () => actions.togglePanel!('topInfluencers', { direction: 'left' });
+
+          return (
+            <>
+              <EuiResizablePanel
+                panelRef={topInfluencersPanelRef}
+                id={'topInfluencers'}
+                mode={[
+                  'collapsible',
+                  {
+                    'data-test-subj': 'mlTopInfluencersToggle',
+                    position: 'top',
+                  },
+                ]}
+                minSize={'200px'}
+                initialSize={20}
+                paddingSize={'s'}
+                onToggleCollapsed={onToggleCollapsed}
+              >
+                {noInfluencersConfigured ? (
```

Comment:
If no influencers are configured is it possible to automatically reduce the width of the panel so it just fits the width of the info icon. Otherwise you are left with a lot of unnecessary space on switching jobs:

![image](https://user-images.githubusercontent.com/7405507/163218345-5298adbc-3701-479b-bca7-ae1dea7d17d9.png)

On switching back to a job with influencers could it then revert to the width saved in local storage?